YogaKit/Source/YGLayout.m

```diff
@@ -89,7 +89,8 @@ - (void)set##objc_capitalized_name:(CGFloat)objc_lowercased_name
 
 @interface YGLayout ()
 
-@property (nonatomic, weak, readonly) UIView *view;
+@property (nonatomic, assign, readonly) YGNodeRef node;
```

Comment:
Can we declare these as ivars in the header? We don't really need the readwrite properties, and it only bloats the memory footprint of these objects.